language/Spanish_Chile/strings.po

```diff
@@ -0,0 +1,6575 @@
+"Last-Translator: MELERIX\n"
+"Language-Team: \n"
+"Language: es_CL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/frontend/mame/ui/ui.cpp:474 src/frontend/mame/ui/ui.cpp:547
+msgid ""
+"\n"
+"\n"
+"Press any key to continue"
+msgstr ""
+"\n"
+"\n"
+"Presiona cualquier tecla para continuar"
+
+#: src/frontend/mame/ui/ui.cpp:559
+msgid ""
+"This system requires media images to be mounted for the following device(s): "
+msgstr ""
+"Este sistema requiere que se monten imágenes de disco para el/los "
+"siguiente(s) dispositivo(s): "
+
+#: src/frontend/mame/ui/ui.cpp:1275
+#, c-format
+msgid ""
+"UI controls enabled\n"
+"Use %1$s to toggle"
+msgstr ""
+"Controles de UI habilitados\n"
+"Usa %1$s para alternar"
+
+#: src/frontend/mame/ui/ui.cpp:1277
+#, c-format
+msgid ""
+"UI controls disabled\n"
+"Use %1$s to toggle"
+msgstr ""
+"Controles de UI deshabilitados\n"
+"Usa %1$s para alternar"
+
+#: src/frontend/mame/ui/ui.cpp:1486
+msgid "Master Volume"
+msgstr "Volumen principal"
+
+#: src/frontend/mame/ui/ui.cpp:1495
+#, c-format
+msgid "%1$s Volume"
+msgstr "Volumen %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1517
+#, c-format
+msgid "Overclock CPU %1$s"
+msgstr "Overclock de CPU %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1525
+#, c-format
+msgid "Overclock %1$s sound"
+msgstr "Overclock de sonido %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1544
+#, c-format
+msgid "%1$s Refresh Rate"
+msgstr "Tasa de refresco %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1549
+#, c-format
+msgid "%1$s Brightness"
+msgstr "Brillo %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1551
+#, c-format
+msgid "%1$s Contrast"
+msgstr "Contraste %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1553
+#, c-format
+msgid "%1$s Gamma"
+msgstr "Gamma %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1557
+#, c-format
+msgid "%1$s Horiz Stretch"
+msgstr "Estiramiento horizontal %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1559
+#, c-format
+msgid "%1$s Horiz Position"
+msgstr "Posición horizontal %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1561
+#, c-format
+msgid "%1$s Vert Stretch"
+msgstr "Estiramiento vertical %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1563
+#, c-format
+msgid "%1$s Vert Position"
+msgstr "Posición vertical %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1579
+#, c-format
+msgid "Laserdisc '%1$s' Horiz Stretch"
+msgstr "Estiramiento horizontal de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1582
+#, c-format
+msgid "Laserdisc '%1$s' Horiz Position"
+msgstr "Posición horizontal de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1584
+#, c-format
+msgid "Laserdisc '%1$s' Vert Stretch"
+msgstr "Estiramiento vertical de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1587
+#, c-format
+msgid "Laserdisc '%1$s' Vert Position"
+msgstr "Posición vertical de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1597
+msgid "Vector Flicker"
+msgstr "Vector de parpadeo"
```

Comment:
"Parpadeo (vector)" - This is talking about the flickering that can be seen in vector graphics.